RELEASE_NOTES.md

```diff
@@ -9,6 +9,13 @@ incompatible changes have been made in that release.
 For a list of all the changes in a particular release, please refer to the changelog file -
 https://github.com/scalyr/scalyr-agent-2/blob/master/CHANGELOG.md.
 
+## 2.1.38 "TBD" - TBD
+
+* This release updates the Kubernetes monitor to more accurately calculate used memory.
```

Comment:
I guess it would be also good to reflect this in the CHANGELOG file. It also seems that it is short enough to just have it in the CHANGELOG file, because, IIRC, we use RELEASE NOTES for more descriptive notes, but nothing wrong with keeping it here too, of course :)

Reply:
Good catch, fixed